### ISSUE TITLE
ability to use two or more providers/ urls within the same container

### ISSUE THREAD

**Issue by aniel300** [hooray4me/ustreamgen#3]
is is possible to use two or more providers/ urls within the same container

**Comment by hooray4me** [comment]
I am working on this

**Comment by hooray4me** [comment]
https://github.com/hooray4me/ustreamgen/tree/EN-multiple-providers
^ this has the latest instructions.
Watch for the lateest *-dev images. currently:

docker pull ghcr.io/hooray4me/ustreamgen:v1.38-dev

One of the issues I need to solve for is how to merge identical shows/movie title from different providers...

Let me know if you think this is going in the right direction.

**Comment by aniel300** [comment]
yes i forgot to mention this. i have couples of ideas. 1 is if there is 1 or more of the same movie then use some kind of numbering, some like.

movie name (year)
movie name (year) (1)
movie name (year) (v1)
and so on

u can decide how to go about this

another idea/question i wanted to ask was about proving the files within the workflow to do some custom file naming using tools like filebot, https://github.com/jkwill87/mnamer, [FFmpeg](https://ffmpeg.org/), etc.

**Comment by hooray4me** [comment]
This becomes a nightmare... because each time it runs, it only copies the files/directories that have changed or are new. and it also removes files/directories if there is no longer a strm file created from the updated m3u from the provider. There would be no way of knowing, on a given run, which of the providers created the movie.

**Comment by aniel300** [comment]
that is sad to hear. so i take it you wont be able to make this happen ?

**Comment by hooray4me** [comment]
I think the only way to potentially make this work is to have a separate folder for each provider... like movies1 movies2 etc... then add those individual folders to emby in the same library.

**Comment by aniel300** [comment]
i can work with that but do u think emby will be happy with that? so about the file probing is no go then as well?

**Comment by hooray4me** [comment]
I've had a scenario where I added multiple folders to the same library. It should be able to handle that

**Comment by aniel300** [comment]
ok that sounds good if not am fine with adding the individual folders to emby. let me know when u have more info/ test image on this.

**Comment by hooray4me** [comment]
I'd be curious, for movies, if it will show the movie twice, or group it into a collection. I have to make some variable changes for that to work. I'll let you know when I have something for you to test.

**Comment by aniel300** [comment]
for same movies there is a plugin that group them
together, i will share it later on

**Comment by hooray4me** [comment]
review the updated readme in this dev branch https://github.com/hooray4me/ustreamgen/tree/EN-multiple-providers on how to setup. Use this image: docker pull ghcr.io/hooray4me/ustreamgen:v1.39-dev

**Comment by aniel300** [comment]
ok will do and report back

**Comment by hooray4me** [comment]
LMK what you find

**Comment by aniel300** [comment]
@hooray4me cant get this to work, please get on discord whenber u can so we can discuss a little more, thanks.
